import-export-cli/cmd/mg/undeployAPI.go

```diff
@@ -60,6 +68,9 @@ var UndeployAPICmd = &cobra.Command{
 		queryParams["apiName"] = undeployAPICmdAPIName
 		queryParams["version"] = undeployAPICmdAPIVersion
 		queryParams["vhost"] = undeployAPICmdAPIVHost
+		// Since other flags does not use args[], gateway-env flag will own all the args
+		gatewayNames := append(args, undeployAPICmdAPIGatewayEnv)
```

Comment:
We can use something like this: https://github.com/spf13/cobra/issues/661#issuecomment-469223372

Reply:
Yes, I also prefer it using -g <env1> -g <env2> -g <env2>
but synapse command is using -g <env1 env2 env3> approach, if we introduce the first approach then we break the inconsistency, that is why i also follow the same.

Reply:
You don't have to use `-g <env1> -g <env2> -g <env2>`. The command here is `--slice c,d,e` in there example. The flag type is `[]string{}`

Reply:
If we are changing it here, we need to fix it in the synapse gateways undeploy command as well. As this command accepts gateways separated by a comma,  synapse gateway command accepts gateways separated by a space. We should keep the consistency.

Reply:
Thanks @SuKSW for pointing this, fixed with https://github.com/wso2/product-apim-tooling/pull/680/commits/ee5f56ebad413e0da8c7af122f204b013e51c4b5
synapse gateway side commands will be fixed with https://github.com/wso2/product-apim-tooling/issues/682